### ISSUE TITLE
Scrolling feels horrible

### ISSUE THREAD

**Issue by darsain** [easymodo/qimgv#199]
When scrolling is bound to a mouse wheel, the faster I scroll, the less it scrolls. Seems like each new scroll event replaces the previous one instead of adding to it.

And if scroll is bound to a key, there is this annoying pause until key stars repeating. I know it's standard key repeat behavior, but it makes no sense for scrolling. And each scroll event also seems to replace the previous one instead of adding to it, though it's not as noticeable here.

**Comment by easymodo** [comment]
you mean scrolling the image or the thumbnails?

if it's the image then yeah, i'm aware of this. will fix

**Comment by darsain** [comment]
Yes, image ❤️  Are you also planning on removing the pause before key starts repeating?

**Comment by easymodo** [comment]
that would be tricky without hardcoding the keys to scroll only.. need think about it

**Comment by easymodo** [comment]
tbh i just lower a key repeat delay in the OS to like 200ms and the problem mostly disappears

**Comment by darsain** [comment]
I guess I'm too used to navigating with [Vimium](https://github.com/philc/vimium). Immediate smooth scrolling when pressing d,u or j,k feels SO good.

**Comment by eugenesvk** [comment]
I'd also like to chime in to ask to have an option to have no delay whatsoever while zooming with a keyboard, that pause before it starts repeating is indeed a bit annoying!
Thank you!
